(.devops) ubuntu@ip-172-31-27-175:~/Projects/Project-4-ml-microservice-kubernetes$ python3 appcopy.py
 * Serving Flask app "appcopy" (lazy loading)
 * Environment: production
   WARNING: Do not use the development server in a production environment.
   Use a production WSGI server instead.
 * Debug mode: on
Traceback (most recent call last):
  File "appcopy.py", line 88, in <module>
    app.run(host='0.0.0.0', port=80, debug=True) # specify port=8081
  File "/home/ubuntu/miniconda3/envs/.devops/lib/python3.6/site-packages/flask/app.py", line 943, in run
    run_simple(host, port, self, **options)
  File "/home/ubuntu/miniconda3/envs/.devops/lib/python3.6/site-packages/werkzeug/serving.py", line 988, in run_simple
    s.bind(server_address)
PermissionError: [Errno 13] Permission denied

what we do is
line 943, in run run_simple
run_simple(host, port, self, **options)
run_simple(host, port, self, debug)

line 988, in run_simple
s.bind(server_address)
#s.bind(server_address)


(.devops) ubuntu@ip-172-31-27-175:~/Projects/Project-4-ml-microservice-kubernetes$ python3 app.py
 * Serving Flask app "app" (lazy loading)
 * Environment: production
   WARNING: Do not use the development server in a production environment.
   Use a production WSGI server instead.
 * Debug mode: on
 * Running on http://0.0.0.0:36693/ (Press CTRL+C to quit)
 * Restarting with stat
127.0.0.1 - - [20/Oct/2021 21:19:00] "GET / HTTP/1.1" 200 -
127.0.0.1 - - [20/Oct/2021 21:19:01] "GET / HTTP/1.1" 200 -
127.0.0.1 - - [20/Oct/2021 21:19:01] "GET /favicon.ico HTTP/1.1" 404 -
127.0.0.1 - - [20/Oct/2021 21:19:02] "GET /favicon.ico HTTP/1.1" 404 -

